snuba/state/__init__.py

```diff
@@ -92,9 +98,13 @@ def wrapper() -> Any:
         return wrapper
 
 
-def numeric(value: Any) -> Any:
+def get_typed_value(value: Any) -> Any:
```

Comment:
Ideally I think this should be:
```suggestion
def get_typed_value(value: str) -> Optional[str, int, float]:
```
I'm not sure why most of this file is just typed as `Any`.